packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -771,6 +771,9 @@ export abstract class SharedObject<
 		id: string,
 		runtime: IFluidDataStoreRuntime,
 		attributes: IChannelAttributes,
+		/**
```

Comment:
I noticed that the other docs in these constructors aren't following TSDoc best practices, so I created a separate PR that adds this.
https://github.com/microsoft/FluidFramework/pull/24495